src/ripple/rpc/handlers/Tx.cpp

```diff
@@ -97,11 +97,12 @@ Json::Value doTx (RPC::Context& context)
     if (!isHexTxID (txid))
         return rpcError (rpcNOT_IMPL);
 
+    int error_code;
```

Comment:
If there is no error then it appears `error_code` remains unitialized. I'd initialize it to `rpcSUCCESS`.